src/main/java/io/github/ust/mico/kafkafaasconnector/kafka/MicoCloudEventImpl.java

```diff
@@ -112,6 +132,10 @@ public void setUnknownExtensions(String key, String value) {
         return this;
     }
 
+    public boolean isFilterOutNecessary(String topic) {
```

Comment:
I will add JavaDoc but could you recommend a better name?

Reply:
Maybe isTestMessageCompleted/Finished because the method is only true for test messages that completed their test purpose and are ready for inspection.